addon/components/detail-learnergroups-list.hbs

```diff
@@ -25,9 +25,9 @@
           {{/if}}
           <ul>
             <li class="top-level-group" data-test-subgroup>
-              {{tree.topLevelGroup.title}} ({{count-related tree.topLevelGroup "users"}})
+              <span data-test-title>{{tree.topLevelGroup.title}} ({{count-related tree.topLevelGroup "users"}})</span>
               {{#if tree.topLevelGroup.needsAccommodation}}
-                <FaIcon @icon="universal-access" title={{t "general.membersOfThisGroupRequireAccommodation"}}></FaIcon>
+                <FaIcon @icon="universal-access" @title={{t "general.membersOfThisGroupRequireAccommodation"}}></FaIcon>
```

Comment:
Ugh. As the maintainer of this addon I'm sorry. It probably ate your original title because it's missing `...attributes`.